src/sidebar/components/SelectionTabs.js

```diff
@@ -42,28 +44,32 @@ function Tab({
   const title = count > 0 ? `${label} (${count} available)` : label;
 
   return (
-    <div>
-      <button
-        className={classnames('SelectionTabs__type', {
-          'is-selected': isSelected,
-        })}
-        // Listen for `onMouseDown` so that the tab is selected when _pressed_
-        // as this makes the UI feel faster. Also listen for `onClick` as a fallback
-        // to enable selecting the tab via other input methods.
-        onClick={selectTab}
-        onMouseDown={selectTab}
-        role="tab"
-        tabIndex={0}
-        title={title}
-        aria-label={title}
-        aria-selected={isSelected.toString()}
-      >
+    <LabeledButton
+      classes={classnames('u-color-text', 'SelectionTab', {
+        'is-selected': isSelected,
+      })}
+      // Listen for `onMouseDown` so that the tab is selected when _pressed_
+      // as this makes the UI feel faster. Also listen for `onClick` as a fallback
+      // to enable selecting the tab via other input methods.
+      onClick={selectTab}
+      onMouseDown={selectTab}
+      pressed={!!isSelected}
+      role="tab"
+      tabIndex={0}
+      title={title}
+    >
+      <div>
```

Comment:
What does this `<div>` do? Could it just be a fragment?

Reply:
Nice catch. Yep, I'm changing to a Fragment.